lib/src/create_report.dart

```diff
@@ -94,7 +91,7 @@ Future<ReportSection> _hasDocumentation(
   final points = examplePath == null ? 0 : 10;
   final status = examplePath == null ? _Status.bad : _Status.good;
   return ReportSection(
-    title: documentationSectionTitle,
+    title: 'Provide documentation',
```

Comment:
Please update the const, it used to merge the separately created coverage subsection.